.aios-core/core/orchestration/workflow-orchestrator.js

```
@@ -0,0 +1,676 @@
+        return { success: true, tool: action.tool };
+
+      case 'check_env':
+        const missing = [];
+        for (const varName of (action.vars || [])) {
+          if (!process.env[varName]) {
+            missing.push(varName);
+          }
+        }
+        if (missing.length > 0 && action.blocking !== false) {
+          throw new Error(`Missing environment variables: ${missing.join(', ')}`);
+        }
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)